src/components/dropdown/dropdown.css

```diff
@@ -0,0 +1,151 @@
+
+:host {
+  --primary-color: #41a6ef;
+  --secondary-color: #fff;
+  --drop-background: var(--dropdown-color, var(--primary-color));
+  --drop-border:  var(--dropdown-border-color, var(--primary-color));
+  --drop-text-color: var(--dropdown-text-color, var(--secondary-color));
+  --drop-text-size:  var(--dropdown-text-size, 1rem);
+  --drop-text-hover-color: var(--dropdown-text-hover-color, var(--secondary-color));
+  --drop-text-expanded-color: var(--dropdown-text-expanded-color, var(--primary-color));
+  --drop-text-background-hover: var(--dropdown-text-background-hover, var(--primary-color));
+  --drop-timing: var(--global--dropdown-timing, .3s);
+
+    height: 2em;
+    position: relative;
+    display: block;
+
+  & .dropdown-el {
+    min-width: 12em;
+    position: relative;
+    display: inline-block;
+    margin-right: 1em;
+    min-height: 2em;
+    max-height:2em;
+    overflow:hidden;
+    font-size: var(--drop-text-size);
+    top: .5em;
+    cursor: pointer;
+    text-align: left;
+    white-space: nowrap;
+    color: var(--drop-text-color);
+
+    outline: none;
+    border: .06em solid transparent;
+    border-radius: 1em;
+    background-color: var(--drop-background);
+
+    transition: var(--drop-timing) all ease-in-out;
+    & input {
+      display:none;
+    }
+    & label {
+      border-top: .06em solid #d9d9d9;
+      display:block;
+      height: 2em;
+      line-height: 2em;
+      padding-left: 1em;
+      padding-right: 3em;
+      cursor: pointer;
+      position: relative;
+      &:nth-child(2) {
+        margin-top: 2em;
+        border-top: .06em solid #d9d9d9;
+      }
+    }
+    /* Display default option first, if not selected/expanded */
+    & #option_def + label {
+      display:block;
+      border-top: none;
+      position: absolute;
+      top: 0;
+
+      &:nth-child(2) {
+        margin-top: 0;
+        position: relative;
+      }
+    }
+    /* If selection made, don't display default, display selected */
+    &.selection {
+      & input + label {
+        margin-top: 2em;
+      }
+
+      & #option_def + label {
+        display:none;
+      }
+
+      & input:checked + label {
+        display:block;
+        border-top: none;
+        position: absolute;
```

Comment:
That's a nasty bug I didn't notice before. The problem is when an item is selected the selected label changes to position:absolute and set to top:0 that's why we see it as the selected option once the dropdown disappears.  But once the absolute position is set, the previous width, which accounted for that label's width, disappears, and now, the next largest item is used to calculate the width(or it defaults to a minimum width of 12em).  

I suppose one way to fix it would be to do change the selected option's label programmatically on click event with javascript instead of using css to position its label at the top.